cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -104,24 +137,24 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 		if args.StreamOptimized {
 			f, err = vmware.OpenAsStreamOptimizedVmdk(imagePath)
 			if err != nil {
+				r = append(r, err)
```

Comment:
These errors are not really `TargetErrors`, I suggest introducing a new `InternalError` and setting that to the first internal error we get (and exiting when that happens).

Reply:
See my comment bellow. I think this is a good idea, but I'd implement it as part of a new PR because this might again turn out to be a bigger issue than just sending a single message.